src/frontend/components/Felleskomponenter/Barnehagemodal/barnehageplassPeriodeOppsummering.tsx

```diff
@@ -34,54 +35,66 @@ export const BarnehageplassPeriodeOppsummering: React.FC<BarnehageplassPeriodePr
     } = barnehageplassPeriode;
 
     const [valgtLocale] = useSprakContext();
-
-    const spørsmålSpråkTekst = (spørsmålId: BarnehageplassPeriodeSpørsmålId) => (
-        <SpråkTekst id={spørsmålId} values={{ barn: barnetsNavn }} />
-    );
+    const { tekster, plainTekst } = useApp();
+    const barnehageplassTekster: IBarnehageplassTekstinnhold =
+        tekster()[ESanitySteg.FELLES].modaler.barnehageplass;
 
     return (
         <PeriodeOppsummering
             fjernPeriodeCallback={
                 fjernPeriodeCallback && (() => fjernPeriodeCallback(barnehageplassPeriode))
             }
-            fjernKnappSpråkId={'todo.ombarnet.barnehageplass.periode'}
+            fjernKnappTekst={barnehageplassTekster.fjernKnapp}
             nummer={nummer}
             tittelSpråkId={'todo.ombarnet.barnehageplass.periode'}
         >
             <OppsummeringFelt
-                tittel={spørsmålSpråkTekst(
-                    BarnehageplassPeriodeSpørsmålId.barnehageplassPeriodeBeskrivelse
+                tittel={plainTekst(barnehageplassTekster.periodebeskrivelse.sporsmal)}
```

Comment:
Tror jeg har det i en PR jeg har ute nå, men den tittelen er egentlig deprecated. Så her kan man sende inn `spørsmålstekst` i stedet for 😊 da får man riktig typografi og alt sånn der